components/script/dom/eventtarget.rs

```diff
@@ -225,7 +228,11 @@ impl CompiledEventListener {
 /// A listener in a collection of event listeners.
 struct EventListenerEntry {
     phase: ListenerPhase,
-    listener: EventListenerType
+    listener: EventListenerType,
+    capture: bool,
+    passive: bool,
+    once: bool,
+    removed: bool,
```

Comment:
Don't add fields for which we don't support the corresponding thing, so just add `capture`.